manifests/lk0001/CurrentTemplates/0.1.1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "manifest_version": "1",
+  "name": "Current Templates",
+  "namespace": "lk0001.CurrentTemplates",
+  "version": "0.1.1",
+  "contributors": [
```

Comment:
Autogenerated manifest has `"contributors": null`, so I copied it from previous manifest.

Reply:
Perfect.  I need to resolve that in the script to help make sure that doesn't happen in the future.  Thanks for the reminder.  👍